### ISSUE TITLE
Add shift support to remote tree

### ISSUE THREAD

**Issue by jimmaaay** [icetee/remote-ftp#492]
Make it so you're able to select multiple files by holding shift.

**Comment by bkonia** [comment]
Agreed. Currently, you can cmd-click to select multiple files one at a time. However, shift-clicking a range of files doesn't work as expected. Can you please add this functionality?
